src/test/java/lotto_tests/LottoMatchCountTests.java

```diff
@@ -0,0 +1,49 @@
+package lotto_tests;
```

Comment:
테스트 클래스는 테스트 대상 클래스와 같은 패키지에 위치하면 어떨까요?

```suggestion
package lotto.model.wrapper;
```